.github/workflows/keyvi.yml

```diff
@@ -0,0 +1,83 @@
+# Build on Linux
+
+name: Build keyvi
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
```

Comment:
Maybe we want to build other branches too ? 
For example release branches.

Reply:
afaik, that requires a separate trigger: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release

That trigger is not based on a branch name, which will have a different name anyway, not sure if you can specify wild cards.

Reply:
From [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#on)
```
# Triggered when code is pushed to any branch in a repository
on: push
```
So if we remove `branches: [ master ]` part, then it will start building everything I guess.